core/mondoo-ms365-security.mql.yaml

```diff
@@ -0,0 +1,625 @@
+         2. Click on "Azure Active Directory" once in the "Admin Centers"
+         3. Applications --> Enterprise applications --> Security --> Conditional Access 
+         4. Make sure there is a Policy with following conditions:
+           * Grant --> Grant access --> Require muilti-factor authentication --> then Select 
+           * Users --> Users and groups --> Include --> All users 
+           * Cloud apps or actions --> All cloud apps 
+           * Conditions --> sign-in risk --> Yes in the right pane followed by the appropriate level.
+         5. Make sure that the policy is Enabled 
+    query: |
+      msgraph.beta.security.latestSecureScores.controlScores.one( _['controlName'] == 'SigninRiskPolicy' && _['score'] == 7 )
+  - uid: mondoo-m365-enable-azure-ad-identity-protection-user-risk-policies
+    title: Enable Azure AD Identity Protection sign-in 
+    severity: 100
+    docs:
+      desc: |
+        This control checks that there are some policies in place which can detect risky sign-in in real-time and offline. A risky sign-in mainly means a sign-in attempt which might be performed by illegitimate owner of a user account.
+      audit: |
+        __cnquery run__
+
+        To audit Microsoft MS365 with `cnquery run`:
+
+        Run the following query:
+
+          ```mql
+          cnquery run ms365 -c "msgraph.beta.security.latestSecureScores.controlScores.one( _['controlName'] == 'UserRiskPolicy' && _['score'] == 7 )" --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+        __cnquery shell__
+
+        To audit Microsoft ms365 with `cnquery shell`:
+
+        1. Launch `cnquery shell`:
+
+          ```bash
+          cnquery shell ms365 --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+        
+        2. Run the following query:
+
+          ```mql
+          msgraph.beta.security.latestSecureScores.controlScores.one( _['controlName'] == 'UserRiskPolicy' && _['score'] == 7 )
+          ```
+      remediation: |
+         ### MS365 Console
+         
+         To config sign-in risk policy via the ms365 portal:
+
+         1. Log in as Global Administrator to the ms365 portal at https://admin.microsoft.com
+         2. Click on "Azure Active Directory" once in the "Admin Centers"
+         3. Applications --> Enterprise applications --> Security --> Conditional Access 
+         4. Make sure there is a Policy with following conditions:
+           * Grant --> Grant access --> Require muilti-factor authentication --> then Select 
+           * Users --> Users and groups --> Include --> All users 
+           * Cloud apps or actions --> All cloud apps 
+           * Conditions --> sign-in risk --> Yes in the right pane followed by the appropriate level.
+         5. Make sure that the policy is Enabled 
+    query: |
+      msgraph.beta.security.latestSecureScores.controlScores.one( _['controlName'] == 'UserRiskPolicy' && _['score'] == 7 )
```

Comment:
`msgraph.beta.security` is deprecated, use latest MQL resource

Reply:
done!